AcadeMeData/tests/test_navbar_links.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from AcadeMeData.models import User
+
+
+@pytest.fixture
+def user_example():
+    user_data = {'username': "username", 'password': "password", 'email': "user@example.com",
+                 'university': "RU", 'degree': "CS"}
+    user = User.create_user(*user_data)
+    return user
+
```

Comment:
maybe this should move to? https://github.com/redhat-beyond/AcadeMe/blob/main/AcadeMeData/conftest.py

or at least open an Issue related to moving user_example over there? and than suggest a new PR?